internal/pkg/libexec/pull.go

```diff
@@ -39,13 +42,18 @@ func PullShubImage(filePath, shubRef string, force, noHTTPS bool) {
 }
 
 // PullOciImage pulls an OCI image to a sif
-func PullOciImage(path, uri string, opts types.Options) {
-	b, err := build.NewBuild(uri, build.Config{Dest: path, Format: "sif", Opts: opts})
+func PullOciImage(path, imageURI string, opts types.Options) {
+	_, imageName := uri.Split(imageURI)
+	if !strings.Contains(imageName, ":") {
```

Comment:
I don't think this is the behavior we want from an `oci` pull. My understanding of the issue was that this should only affect pulls from `library`